forc-plugins/forc-client/src/constants.rs

```diff
@@ -15,6 +16,7 @@ pub const DEVNET_FAUCET_URL: &str = "https://faucet-devnet.fuel.network";
 pub const TESTNET_FAUCET_URL: &str = "https://faucet-testnet.fuel.network";
 
 pub const TESTNET_EXPLORER_URL: &str = "https://app.fuel.network";
```

Comment:
I think we can change this into:

```suggestion
pub const TESTNET_EXPLORER_URL: &str = "https://app-tesnet.fuel.network";
```

Reply:
There is a spelling mistake in your suggestion there @kayagokalp , should be `https://app-testnet.fuel.network` .... testnet was missing a 't'

Reply:
i added this in #6634